src/Microsoft.XmlSerializer.Generator/pkg/build/Microsoft.XmlSerializer.Generator.targets

```diff
@@ -5,29 +5,54 @@
+    <Csc Condition="Exists('$(_SerializerCsIntermediateFolder)') AND Exists('$(_CscRspFilePath)')" ContinueOnError="true" OutputAssembly="$(_SerializerDllIntermediateFolder)" References="@(ReferencePath);@(IntermediateAssembly)" EmitDebugInformation="$(DebugSymbols)" Sources="$(_SerializerCsIntermediateFolder)" TargetType="Library" ResponseFiles="$(_CscRspFilePath)"  ToolExe="$(CscToolExe)" ToolPath="$(CscToolPath)" DisabledWarnings="$(_SerializationAssemblyDisabledWarnings)"/>
     <Warning Condition="Exists('$(_SerializerDllIntermediateFolder)') != 'true' And Exists('$(_SerializerCsIntermediateFolder)') == 'true'" Text="$(_SGenWarningText)"/>
     <Copy Condition="Exists('$(_SerializerDllIntermediateFolder)') == 'true'" SourceFiles="$(_SerializerDllIntermediateFolder)" DestinationFolder="$(OutputPath)" />
+    <Delete Condition="Exists('$(_SgenRspFilePath)')" Files="$(_SgenRspFilePath)" />
+    <Delete Condition="Exists('$(_CscRspFilePath)')" Files="$(_CscRspFilePath)" />
+  </Target>
+
+  <Target Name="GenerateRspFiles" BeforeTargets="GenerateSerializationAssembly">
+    <PropertyGroup>
+      <SGenReferences Condition=" '$(SGenReferences)' == ''">@(ReferencePath)</SGenReferences>
+    </PropertyGroup>
+    <ItemGroup>
+      <SgenRspFile Include ="--reference $(SGenReferences.Replace(';', '%3B'))" />
+      <SgenRspFile Include ="--type $(SGenTypes.Replace(';', '%3B'))" Condition=" '$(SGenTypes)' != ''" />
+      <SgenRspFile Include ="--verbose" Condition=" '$(SGenVerbose)' == 'true'" />
+      <SgenRspFile Include ="--proxytypes" Condition=" '$(SGenProxyTypes)' == 'true'" />
+      <CscRspFile Include="-keyfile:$(SGenKeyFile)" Condition=" '$(SGenKeyFile)' != ''" />
+      <CscRspFile Include="-keycontainer:$(SGenKeyContainer)" Condition=" '$(SGenKeyContainer)' != ''" />
+      <CscRspFile Include ="-delaysign" Condition=" '$(SGenDelaySign)' == 'true'" />
+    </ItemGroup>
+    <WriteLinesToFile File="$(_SgenRspFilePath)" Lines="@(SgenRspFile, '%0A')" Overwrite="true" ContinueOnError="true" />
+    <Warning Condition="!Exists('$(_SgenRspFilePath)')" Text="$(_SGenRspWarningText)" />
+    <WriteLinesToFile Condition="'@(CscRspFile)'!=''" File="$(_CscRspFilePath)" Lines="@(CscRspFile, '%0A')" Overwrite="true" ContinueOnError="true" />
   </Target>
-  
+
   <Target Name="CleanSerializationAssembly" AfterTargets="CoreClean">
     <Message Text="Cleaning serialization files..." Importance="normal"/>
     <Delete Condition="Exists('$(OutputPath)\$(_SerializationAssemblyName).dll') == 'true'" Files="$(OutputPath)\$(_SerializationAssemblyName).dll" />
   </Target>
-  
+
   <Target Name="CopySerializationAssembly" AfterTargets="PrepareForPublish">
```

Comment:
Nit, you've added an extra blank line

Reply:
This is not an extra blank line. After this code change there is still only one blank line between two targets.

Reply:
Odd, it's showing as two blank lines in the code review tool I'm using. I'll open a bug against that tool!